src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -84,7 +84,7 @@ def runTests(name, suffix) {
   try {
     sh 'mkdir -p test-output'
     sh 'chmod 777 test-output'
-    sh "docker-compose -p $name-$suffix-$containerTag -f docker-compose.yaml -f docker-compose.test.yaml up $name"
+    sh "docker-compose -p $name-$suffix-$containerTag -f docker-compose.yaml -f docker-compose.test.yaml up --exit-code-from $name"
```

Comment:
From reading the documentation, you need to pass the service name you want the exit code from, as well as tell it what service you want to bring up, so I think the command would have to be
`up --exit-code-from $name $name`

Reply:
It still works as in this case the $name is superfluous as part of the `up`. 

Most docker compose files only contain one service, or if there is more than service they are linked via 'depends_on' so there is no difference between specify a service name and no services, as they will all start anyway.

I'm inclined to leave it without the second `$name` for sake of simplicity.

Reply:
changed to use `run` rather than `up` following a slack discussion